hack/verify-prometheus-imports.sh

```diff
@@ -37,6 +37,7 @@ source "${KUBE_ROOT}/hack/lib/util.sh"
 # See: https://github.com/kubernetes/kubernetes/issues/89267
 allowed_prometheus_importers=(
   ./cluster/images/etcd-version-monitor/etcd-version-monitor.go
+  ./pkg/kubelet/server/server.go
```

Comment:
The alternative would be to add this new blocking registry type to component-base, but since we don't want to do that anywhere else and we will eventually deprecate cAdvisor container metrics, I think we may as well add an exception if we go this route.